tests/remote_attestation/maa/sgxlkl_enclave/oe_private/oe_header.h

```diff
@@ -0,0 +1,211 @@
+#ifndef OE_HEADER_H
```

Comment:
This file should go away if we expose the cert-related APIs through OE host verify.